tests/parquet_test.py

```diff
@@ -120,3 +120,35 @@ def test_get_datasets(self):
 
         for f in glob.glob('pq_test*'):
             os.remove(f)
+
+    @pytest.mark.optional_parquet
+    def test_against_standard_files(self):
+        datadir = 'resources/parquet-testing'
+        filenames = ['alltypes_plain.parquet',
+                     'alltypes_plain.snappy.parquet',
+                     'delta_byte_array.parquet']
+        columns1 = ['id',
+                    'bool_col',
+                    'tinyint_col',
+                    'smallint_col',
+                    'int_col',
+                    'bigint_col',
+                    'float_col',
+                    'double_col',
+                    'date_string_col',
+                    'string_col']
+        columns2 = ['c_customer_id',
+                    'c_salutation',
+                    'c_first_name',
+                    'c_last_name',
+                    'c_preferred_cust_flag',
+                    'c_birth_country',
+                    'c_login',
+                    'c_email_address',
+                    'c_last_review_date']
+        for basename, ans in zip(filenames, (columns1, columns1, columns2)):
```

Comment:
just want to double check that the repeating of `columns1` is intended

Reply:
After looking at this more, I realize the `columns1` is supposed to be repeated. it corresponds with the file being read. Both `'alltypes_plain.parquet'` and `'alltypes_plain.snappy.parquet'` expect `columns1`. And `'delta_byte_array.parquet'` expects `columns2`